test/client/data_test.rb

```diff
@@ -92,4 +92,11 @@ def test_data_update_custom_field
       assert_true result
     end
   end
+
+  def test_data_get_app_setting
```

Comment:
So some weirdness I ran into on this test: https://travis-ci.org/TheMetalCode/infusionsoft/builds/72658457

I knew something was extra fishy when I was unable to reproduce the error manually. For the Rails service where I'm currently using this gem, I simply changed the ruby version to 1.9.3 and ran Infusionsoft.data_get_app_setting from the rails console, without error.

When I had the bright idea of just setting the content-length of the return to 143 directly in the VCR cassette (https://github.com/TheMetalCode/infusionsoft/commit/8556e4d462ebefeb0b7bb7493f0235566b7a4705), the test was fixed and the latest build passed in all ruby versions.

Because I was unable to repro manually I'm not exactly losing sleep over this, but it's weird to me that the XMLRPC client would care about the content-length of the return in some ruby versions but not others. This is one potential clue: https://bugs.ruby-lang.org/issues/8182